composer.json

```diff
@@ -14,14 +14,14 @@
         }
     ],
     "require": {
-        "php": "^5.5 || ^7.0",
+        "php": "^5.6 || ^7.0",
```

Comment:
Ah, I got your point. Unfortunately, 2.0 version of fos/http-cache brings few BC breaks :( For example, it dropped `...Interface` suffix on all interface classes and a few other changes.